src/ci/docker/x86_64-gnu-tools/checktools.sh

```diff
@@ -97,6 +100,7 @@ status_check() {
 status_check "submodule_changed"
 
 CHECK_NOT="$(readlink -f "$(dirname $0)/checkregression.py")"
+# This callback is called by `commit_toolstate_change`, see `repo.sh`.
 change_toolstate() {
```

Comment:
@kennytm given that this does *not* actually submit a changed toolstate to https://github.com/rust-lang-nursery/rust-toolstate, isn't the same here misleading? Or am I misreading the code again?

Reply:
Hm okay I think I misread. But still, what this does is put the changed toolstate into `history/$OS.tsv`. Then `repo.sh` takes care of the actual publishing.